packages/abi-gen-templates/partials.py/call.handlebars

```diff
@@ -0,0 +1,44 @@
+
+    def {{this.langLocalName}}(
+        self,
+        token_address: str,
+        {{> typed_params inputs=inputs}}
+        {{^this.constant}}
+        tx_params: Optional[TxParams] = None,
+        view_only: bool = False,
```

Comment:
Can you open an issue to refactor the templates in a subsequent PR to use the nested methods setup TS templates use? In addition to `call` vs. `tx`, we will also want to auto-generate methods that return the encoded `calldata` and the `gasEstimation`.

Reply:
Done. #1893